rules/docker-team.mk

```diff
@@ -2,6 +2,6 @@
 
 DOCKER_TEAM = docker-team.gz
 $(DOCKER_TEAM)_PATH = $(DOCKERS_PATH)/docker-team
-$(DOCKER_TEAM)_DEPENDS += $(SWSS)
+$(DOCKER_TEAM)_DEPENDS += $(SWSS) $(LIBTEAM) $(LIBTEAMDCT) $(LIBTEAM_UTILS)
```

Comment:
swss already has a runtime dependency on team
https://github.com/Azure/sonic-buildimage/blob/master/rules/swss.mk#L7
This is redundant